bsonnumpy/bson/bson-config.h

```diff
@@ -123,5 +142,14 @@
 # undef BSON_HAVE_SYSCALL_TID
 #endif
 
+#define BSON_HAVE_RAND_R 1
+#if BSON_HAVE_RAND_R != 1
+# undef BSON_HAVE_RAND_R
+#endif
+
+#define BSON_HAVE_STRLCPY 1
```

Comment:
Let's change this to `0` to fix the travis errors:
```
ERROR: test (unittest.loader._FailedTest)
----------------------------------------------------------------------
ImportError: Failed to import test module: test
Traceback (most recent call last):
  File "/opt/python/3.7.1/lib/python3.7/unittest/loader.py", line 154, in loadTestsFromName
    module = __import__(module_name)
  File "/home/travis/build/mongodb/bson-numpy/test/__init__.py", line 6, in <module>
    import bsonnumpy
  File "/home/travis/build/mongodb/bson-numpy/bsonnumpy/__init__.py", line 2, in <module>
    from ._cbsonnumpy import *
ImportError: /home/travis/build/mongodb/bson-numpy/bsonnumpy/_cbsonnumpy.cpython-37m-x86_64-linux-gnu.so: undefined symbol: strlcpy
```

Reply:
Done.